nevooronni/Politico_web_app
# #163602022 A user can view all political candidates

#### What does this PR do?
This is a PR for a feature that allows a user to view all political candidates running for a particular office

#### Description of Task to be completed?
The following are the tasks to be completed
* add html for this page
* style html markup with css
* Add javascript for modal functionality

#### How to manually test feature?
1. clone this repo
``https://github.com/nevooronni/Politico-Andela.git```

2. cd in folder
```cd Politico-Andela```

3 switch to feature branch 
```git checkout ft-view-politicians-163602022```

4. Open UI/html/index.html in your browser
.
5. Click the signup button on landing page
6. Should be able to view signup page where you will see a signup up form
7. On the bottom of the form click on the signin link
8. You should see a signin page click on the sigin now link to be taken to admin page
9. Click on the user link below profile picture
10. Should be taken to a user view page where a user is able to view all political parties
11. Click on the candidates icon link 
12. should see a page with all political candidates listed

#### Pivotal Tracker story associated with this PR:
story id: #163602022

#### Screenshots
![screenshot from 2019-02-02 21-56-59](https://user-images.githubusercontent.com/20374625/52168063-fba2a100-2735-11e9-9273-7756d4ecfa78.png)
![screenshot from 2019-02-02 21-57-12](https://user-images.githubusercontent.com/20374625/52168064-01988200-2736-11e9-80aa-db78eeb6478f.png)
